client/src/contexts/AppContext.ts

```diff
@@ -32,8 +27,8 @@ export const getElection = ({
   elections,
   electionId,
 }: {
-  elections: ElectionsListItem[]
+  elections: MailElection[]
   electionId: string
-}) => elections.find((e) => e.id === electionId)!
+}) => elections.find(e => e.id === electionId)!
```

Comment:
Do we have competing formatters going on?

Reply:
Hmmmm… will investigate.